include/nigiri/rt/gtfsrt_update.h

```diff
@@ -35,8 +36,16 @@ struct statistics {
   int alert_agency_id_not_found_{0};
   int alert_invalid_route_type_{0};
   int unsupported_deleted_{0};
-  int unsupported_vehicle_{0};
   int unsupported_no_trip_id_{0};
+  int no_vehicle_position_{0};
+  int vehicle_position_without_position_{0};
+  int vehicle_position_without_timestamp_{0};
+  int vehicle_position_without_trip_{0};
+  int vehicle_position_trip_without_trip_id_{0};
+  int duplicate_vehicle_position_timestamps_{0};
+  int outdated_or_illegal_vehicle_position_timestamps_{0};
+  int illegal_stop_{0};
+  int vehicle_position_position_not_at_stop_{0};
```

Comment:
prefix all with vehicle_position_ if they are only relevant for vehicle position updates